packages/node/README.md

```diff
@@ -15,31 +15,49 @@ npm install readmeio
 
 ## Usage
 
-Just add the middleware to express, and that's it!
+Just add the middleware to [Express](https://expressjs.com/), and that's it!
 
 ```javascript
 const readme = require('readmeio');
 
 app.use(readme.metrics('<<apiKey>>', req => ({
-  id: req.project._id,
-  label: req.project.name,
-  email: req.project.owner
+  id: req.<userId>,
+  label: req.<userNameToShowInDashboard>,
+  email: req.<userEmailAddress>,
 })));
 ```
 
 View full documentation here: [https://docs.readme.com/docs/sending-logs-to-readme-with-nodejs](https://docs.readme.com/docs/sending-logs-to-readme-with-nodejs)
 
+### Configuration Options
```

Comment:
Since the PHP SDK is still packaged behind a Laravel service provider, I'm not going to bother with documenting these options, or `baseLogUrl`, in the PHP readme since those docs live in the `config.dist.php` file that's pulled into the users app.